ibis/backends/__init__.py

```diff
@@ -473,6 +473,45 @@ def to_parquet(
                 for batch in batch_reader:
                     writer.write_batch(batch)
 
+    @util.experimental
+    def to_parquet_dir(
+        self,
+        expr: ir.Table,
+        path: str | Path,
```

Comment:
```suggestion
        directory: str | Path,
```

Seems less ambiguous (even though the method name ends with `_dir`).

Reply:
@cpcloud if we do this change which I think it makes sense, do we want to be consistent with the pyspark implementation and changed the name of it in there too?

Reply:
Yep, but both of those are fine to do in a follow-up.